tests/Unit/ParallelAlgorithms/Interpolation/Test_InterpolationTargetSphere.cpp

```diff
@@ -81,47 +125,58 @@ void test_interpolation_target_sphere(
   TestHelpers::db::test_simple_tag<
       intrp::Tags::Sphere<MockMetavariables::InterpolationTargetA>>("Sphere");
 
-  const auto expected_block_coord_holders = [&domain_creator, &radius, &center,
-                                             &angular_ordering]() {
+  const auto expected_block_coord_holders = [&domain_creator, &radii, &center,
+                                             &angular_ordering,
+                                             &number_of_spheres]() {
     // How many points are supposed to be in a Strahlkorper,
     // reproduced here by hand for the test.
     const size_t n_theta = l_max + 1;
     const size_t n_phi = 2 * l_max + 1;
 
-    // The theta points of a Strahlkorper are Gauss-Legendre points.
-    const std::vector<double> theta_points = []() {
-      std::vector<double> thetas(n_theta);
-      std::vector<double> work(n_theta + 1);
-      std::vector<double> unused_weights(n_theta);
-      int err = 0;
-      gaqd_(static_cast<int>(n_theta), thetas.data(), unused_weights.data(),
-            work.data(), static_cast<int>(n_theta + 1), &err);
-      return thetas;
-    }();
-
-    const double two_pi_over_n_phi = 2.0 * M_PI / n_phi;
-    tnsr::I<DataVector, 3, Frame::Inertial> points(n_theta * n_phi);
+    // Have to turn this into a set to guarantee ordering
+    std::set<double> radii_set;
+    for (const double radius : radii) {
+      radii_set.emplace(radius);
+    }
```

Comment:
Simpler: `std::set<double> radii_set(radii.begin(), radii.end());`.  Can probably be const, then.